packages/theme-toolkit/src/style-dictionary.test.ts

```diff
@@ -13,7 +13,7 @@ const fileExists = async (path: string) => {
   }
 };
 const testDirectory = (distPath: string) => {
-  describe('Style Dictionary build', () => {
+  xdescribe('Style Dictionary build', () => {
```

Comment:
TIL dat dit een alias is voor `describe.skip`. That said, ik zou er `describe.skip` van maken.